lit_gpt/lora.py

```diff
@@ -120,8 +120,8 @@ def __init__(
 
         # Actual trainable parameters
         if r > 0:
-            self.lora_A = nn.Parameter(self.linear.weight.new_zeros((r, in_features)))
```

Comment:
Are these changes needed? The previous way is not wrong, right?

Reply:
They fix https://github.com/Lightning-AI/lit-gpt/issues/425